src/components/frame/v5/pages/UserProfilePage/consts.ts

```diff
@@ -1,3 +1,11 @@
-export const MAX_DISPLAYNAME_CHARS = 30;
-export const MAX_BIO_CHARS = 200;
-export const MAX_LOCATION_CHARS = 200;
+import {
+  USER_ADVANCED_ROUTE,
+  USER_EDIT_PROFILE_ROUTE,
+  USER_PREFERENCES_ROUTE,
+} from '~routes/routeConstants';
+
+export const tabRoutes = [
+  USER_EDIT_PROFILE_ROUTE,
+  USER_PREFERENCES_ROUTE,
+  USER_ADVANCED_ROUTE,
+];
```

Comment:
This relies on the order of things matching the arbitrary tab indexes and is very easy to break, could you change it into a map between `TabId` and a corresponding route?